### PR TITLE
feat: started boiler plate

### PR DESCRIPTION
**Pull request by mingkaili**
### Administrative Info
Monday Board ID: 869757475
Make sure to thoroughly read the Pull Request, and make any comments you may have, or request changes on the lines of code. 

### Changes
What changes did you make?
- Added boiler plate + initial files like .gitignore, .env_sample, etc

### Testing
How did you confirm your changes worked? 
- Self checked

### Confirmation of Change 
Confirmed by self check

| [Main Pge screen shot] |
Screen Shot 2020-11-24 at 10 55 43 AM
| [no Link] |

### CODE REVIEW

**Review by jacobayu — APPROVED** [review]

**Review by shravan1k — APPROVED** [review]

File: app/views/index.ejs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title><%= title %></title>
+    </head>
+    <body>
+        <h1>OG YOGA</h1>
```
Comment (hahsu) [inline app/views/index.ejs]: nit: in index.js, you have a dash "OG-YOGA", but here you do not have a dash. Maybe you should pick one to be consistent.

Comment (mingkaili) [reply app/views/index.ejs]: Very good comment, consistency is super important

**Review by deepanshasingh — APPROVED** [review]

File: app.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
+server.on('error', onError);
+server.on('listening', onListening);
+
+automation.startScript();
+
+function onError(error) {
```
Comment (wes-chen) [inline app.js:25]: do you plan on writing javadoc for all functions? if so, did you miss one here?

Comment (mingkaili) [reply app.js:25]: Yes, very good comment, I will add the javadoc comment. It is very important to keep the consistency, and javadoc is also important for most functions(especially the ones already being written)

**Review by eliasfang — APPROVED** [review]

**Review by nvanny — APPROVED** [review]

**Review by skaiwu — APPROVED** [review]

**Pull request merged**